### DISCUSSION TITLE
anyone else having a skin issue

### DISCUSSION THREAD

**Discussion by Shoutmon2** [ppy/osu#19395] [Q&A] [answered]
Put In short no matter the skin i use in osu!lazer after resetting version i keep getting the really old default osu! skin no matter if its the koifishu skin cookeize or BTMC or even my own skin it always goes back to the old 2008 osu default skin

**Comment by Theighlin** [comment, marked as answer]
Read #15188 and update your post with logs and a video showing the problem.
